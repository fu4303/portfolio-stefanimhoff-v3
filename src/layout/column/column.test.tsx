import React from 'react';

import { render } from '../../services/test-utils';

import { Column } from '.';

describe('Column', () => {
  test('renders correctly', () => {
    const { container } = render(<Column>Column</Column>);

    expect(container.firstChild).toMatchSnapshot();
  });

  test('with "fullsize" renders correctly', () => {
    const { container } = render(<Column size="fullsize">Column</Column>);

    expect(container.firstChild).toMatchSnapshot();
  });

  test('with "regular" renders correctly', () => {
    const { container } = render(<Column size="regular">Column</Column>);

    expect(container.firstChild).toMatchSnapshot();
  });

  test('with "narrow" renders correctly', () => {
    const { container } = render(<Column size="narrow">Column</Column>);

    expect(container.firstChild).toMatchSnapshot();
  });
});
