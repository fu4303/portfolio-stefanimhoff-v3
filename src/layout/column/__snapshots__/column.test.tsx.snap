// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Column renders correctly 1`] = `
.c0 {
  grid-column: 1 / -1;
  width: 100%;
}

<div
  class="c0"
>
  Column
</div>
`;

exports[`Column with "fullsize" renders correctly 1`] = `
.c0 {
  grid-column: 1 / -1;
  width: 100%;
}

<div
  class="c0"
>
  Column
</div>
`;

exports[`Column with "narrow" renders correctly 1`] = `
.c0 {
  grid-column: 2 / -2;
  width: 100%;
}

@media (min-width:768px) {
  .c0 {
    grid-column: 6 / -6;
  }
}

@media (min-width:1200px) {
  .c0 {
    grid-column: 7 / -7;
  }
}

<div
  class="c0"
>
  Column
</div>
`;

exports[`Column with "regular" renders correctly 1`] = `
.c0 {
  grid-column: 2 / -2;
  width: 100%;
}

@media (min-width:768px) {
  .c0 {
    grid-column: 3 / -3;
  }
}

<div
  class="c0"
>
  Column
</div>
`;
